lib/ansible/plugins/doc_fragments/acme.py

```diff
@@ -76,6 +78,9 @@ class ModuleDocFragment(object):
       - "For safety reasons the default is set to the Let's Encrypt staging
          server (for the ACME v1 protocol). This will create technically correct,
          but untrusted certificates."
+      - "The default value is U(https://acme-staging.api.letsencrypt.org/directory).
```

Comment:
Perhaps "By default this module uses the staging ACME directory." Then provide the full URL in the `default:` line.

Reply:
See above.